seahorn/lib/nd_klee.c

```diff
@@ -45,3 +117,23 @@ void __VERIFIER_error(void) {
   klee_assert(0);
 }
 
+/* memhavoc - Special function for memory block initialization. 
+  * initialize memory content as non-deterministic value.
+*/
+void memhavoc(void *ptr, size_t size) {
+  for (size_t i = 0; i < size; ++i){
```

Comment:
NO!!!!!!

Reply:
Please check my new implementation. Let me know if it is still wrong.